mu-trees/addon/resolvers/glimmer-wrapper/index.js

```diff
@@ -24,21 +24,35 @@ const Resolver = DefaultResolver.extend({
 
   normalize: null,
 
-  resolve(lookupString) {
+  resolve(lookupString, referrer) {
     /*
-     * Ember paritals are looked up as templates. Here we replace the template
+     * Ember partials are looked up as templates. Here we replace the template
      * resolution with a partial resolute when appropriate. Try to keep this
      * code as "pay-go" as possible.
      */
 
     if (lookupString.indexOf('template:') === 0) {
       lookupString = this._templateToPartial(lookupString);
     }
-    return this._resolve(lookupString);
+    return this._resolve(lookupString, referrer);
   },
 
-  _resolve(lookupString) {
-    return this._glimmerResolver.resolve(lookupString);
+  _resolve(lookupString, referrer) {
+    if (referrer) {
+      // make absolute
+      let appName = this.config.app.name;
+      let parts = referrer.split(':src/ui/');
+      referrer = `${parts[0]}:/${appName}/${parts[1]}`;
+      referrer = referrer.split('/template.hbs')[0];
```

Comment:
An aside, but @dgeb thinks @wycats and @chancancode are perhaps working on adding the absolute specifier to the template meta a compilation time. One that work lands we could change Ember to pass that and this code goes away 👍